instaloader/__main__.py

```diff
@@ -128,6 +153,7 @@ def _main(instaloader: Instaloader, targetlist: List[str],
     anonymous_retry_profiles = set()
     try:
         # Generate set of profiles, already downloading non-profile targets
+        targetlist = targetlist_parser(targetlist) # Removes Blanklinks
```

Comment:
This does not seem clean to me. What is the advantage of changing the parsed targets, rather than overriding [ArgumentParser.convert_arg_line_to_args()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.convert_arg_line_to_args)?